cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateDatasetInstanceCommand.java

```diff
@@ -52,21 +53,32 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     String datasetName = arguments.get(ArgumentName.NEW_DATASET.toString());
     String datasetPropertiesString = arguments.getOptional(ArgumentName.DATASET_PROPERTIES.toString(), "");
     String datasetDescription = arguments.getOptional(ArgumentName.DATASET_DESCRIPTON.toString(), null);
+    String datasetOwner = arguments.getOptional(ArgumentName.OWNER_PRINCIPAL.toString(), null);
```

Comment:
We use `principal` in other commands, such as `create namespace <namespace-name> [description <description>] [principal <principal>]...` though if it is different than `principal`, `owner` is fine. 

See for a list of current commands and syntax: http://docs.cask.co/cdap/current/en/reference-manual/cli-api.html#available-commands